src/styles/container.scss

```diff
@@ -0,0 +1,10 @@
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.